src/resolvers/git.cr

```diff
@@ -100,12 +100,25 @@ module Shards
       "git"
     end
 
+    private KEY_HOSTS = {"www.github.com", "github.com", "www.bitbucket.com", "bitbucket.com", "www.gitlab.com", "gitlab.com"}
```

Comment:
```suggestion
    private CASE_INSENSITIVE_HOSTS = {"www.github.com", "github.com", "www.bitbucket.com", "bitbucket.com", "www.gitlab.com", "gitlab.com"}
```

Reply:
I prefer key hosts, as we are matching the hosts against the key sources

Reply:
Maybe `KNOWN_PROVIDERS` would be a better alternative?